### PR TITLE
visual updates to SuggestionCard

### PR DESCRIPTION
**Pull request by twoodmanATK**

### CODE REVIEW

File: src/components/Cards/SuggestionCard/components/SuggestionCardDek.js [hunk 1]
```diff
@@ -5,9 +5,13 @@ import { spacing } from '../../../../styles';
 const SuggestionCardDek = styled.div.attrs({
   className: 'suggestion-card__dek',
 })`
+  display: -webkit-box !important;
+  -webkit-line-clamp: 3;
+  -webkit-box-orient: vertical;
```
Comment (harry-atk) [inline src/components/Cards/SuggestionCard/components/SuggestionCardDek.js:10]: Mixin exists for this in styles/mixins/truncateLineClamp

File: src/components/Cards/SuggestionCard/components/SuggestionCardActions.js [hunk 2]
```diff
@@ -14,12 +14,12 @@ const SuggestionCardActions = styled.div.attrs({
   }
 
   @media (min-width: 0) and (max-width: 413px) {
-    flex-direction: column;
+    flex-direction: column-reverse;
```
Comment (harry-atk) [inline src/components/Cards/SuggestionCard/components/SuggestionCardActions.js:17]: The reverse directions have accessibility issues. https://developer.mozilla.org/en-US/docs/Web/CSS/flex-direction#accessibility_concerns. I don't know if there is an easy fix for this.

Comment (harry-atk) [reply src/components/Cards/SuggestionCard/components/SuggestionCardActions.js:17]: You might have to use src/components/hooks/useMedia.js to change dom ordering.

Comment (twoodmanATK) [reply src/components/Cards/SuggestionCard/components/SuggestionCardActions.js:17]: 👍

Comment (twoodmanATK) [reply src/components/Cards/SuggestionCard/components/SuggestionCardActions.js]: Molly and I had a chat and we are just swapping the elements' order in the jsx, she agreed that it made sense for all breakpoints

**Review by ttornese — APPROVED** [review]

**Pull request merged**